processor/spanmetricsprocessor/config_test.go

```diff
@@ -36,13 +36,21 @@ import (
 func TestLoadConfig(t *testing.T) {
 	defaultMethod := "GET"
```

Comment:
`t.Parallel()` please

Reply:
As this code is not introduced in this PR, let's do that in a separate PR?